### PR TITLE
Converting timeIM into EquationalModel

### PR DESCRIPTION
**Pull request by Awurama-N**
contributes to Issue #2569

### CODE REVIEW

File: code/drasil-example/Drasil/Projectile/IMods.hs [hunk 1]
```diff
@@ -33,17 +33,18 @@ import Drasil.Projectile.Unitals (flightDur, landPos, launAngle, launSpeed,
 iMods :: [InstanceModel]
 iMods = [timeIM, landPosIM, offsetIM, messageIM]
 
+iMods0 :: [InstanceModel]
+iMods0 = [landPosIM, offsetIM, messageIM]
```
Comment (balacij) [inline code/drasil-example/Drasil/Projectile/IMods.hs:37]: We need to make sure that these are the only IMs for which we use `relToQD` on as well, in the Body.hs

File: code/drasil-example/Drasil/Projectile/IMods.hs [hunk 2]
```diff
@@ -33,17 +33,18 @@ import Drasil.Projectile.Unitals (flightDur, landPos, launAngle, launSpeed,
 iMods :: [InstanceModel]
 iMods = [timeIM, landPosIM, offsetIM, messageIM]
```
Comment (balacij) [inline code/drasil-example/Drasil/Projectile/IMods.hs:34]: Adding to the below comment, we also need to use the "getEqModQdsFromIMs" function on this in the Body.hs -- exact name is on the other PR.

**Review by balacij — CHANGES_REQUESTED** [review]

**Comment by Awurama-N** [comment]
if this is okay, can i go ahead and do the next ones? @balacij

**Comment by balacij** [comment]
Yep, go ahead and start working on the next one in a new branch based on this one. When that one is ready, create a new PR for it and note that it depends on this PR (assuming this one hasn't been merged by then).

**Comment by balacij** [comment]
You should make sure to branch off of this `convProjectileTimeIM` branch, otherwise there may be a merge conflict. By keeping the existing commits on your new branch and "depending" on it or this PR, you're ensuring that merge conflicts wont occur as a result (unless something else gets merged into master that conflicts with this original branch).

**Review by JacquesCarette — APPROVED** [review]

**Comment by JacquesCarette** [comment]
@balacij as you asked for changes, your explicit approval, through a review, is needed for github to be happy to merge.

**Review by balacij — APPROVED** [review]

**Comment by balacij** [comment]
@JacquesCarette That's interesting, I wasn't aware of that. Thank you.

**Pull request merged**